### PR TITLE
Description of the package

### PR DESCRIPTION
**Pull request by vcuspinera**

### CODE REVIEW

**Review by jackyho112 — COMMENTED** [review]
Thanks for adding the other docs!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
+
+## Contributing
```
Comment (jackyho112) [inline CONTRIBUTING.md:2]: A lot of content here only applies to the Python package. Could copy this [one](https://github.com/jackyho112/DSCI_522_Group405/blob/master/CONTRIBUTING.md) instead?

**Review by jackyho112 — COMMENTED** [review]
Remember to copy over the changes you have at the Python version.

File: CONTRIBUTING.md (hunk 1 above)
Comment (vcuspinera) [reply CONTRIBUTING.md:2]: Done!

**Review by dkruszew — APPROVED** [review]
Looks good

**Pull request merged**